hack/dockerfiles/test.Dockerfile

```diff
@@ -50,7 +51,29 @@ WORKDIR /go/src/github.com/moby/buildkit
 COPY . .
 
 FROM cross-windows AS buildctl.exe
+ARG GOOS=windows
```

Comment:
what is this?

Reply:
ah, sorry. I missed that it was already defined.

Reply:
@AkihiroSuda updated